### PR TITLE
Added and styled the sections on youtube.html and added images to the homepage

### PR DESCRIPTION
**Pull request by jonas-45**
Youtube-clone has a homepage with multiple sections on the homepage. The sections are styled with images to show the lists of videos available on the Youtube page. It has an overflow icon to the top-left  with a search bar at the top centre of the homepage.

### CODE REVIEW

File: youtube.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            <button id="microphone">
+                <img id="voice-icon" src="icons/microphone.png" alt="">
+            </button>
+        </div>
+        <div id="right-session">
+            <img id="upload-icon" src="icons/upload.png" alt="">
+            <img id="video-icon" src="icons/video-player.png" alt="">
+            <img id="bell-notification" src="icons/bell.png" alt="">
+            <img id="profile-pic" src="profiles/channel-pic.jpg" alt="">
+        </div>
```
Comment (jonas-45) [inline youtube.html:56]: Please consider using `<ul>` and `<li>` tags to display the items. Using `<ul>` and `<li>` tags gives meaning to the html nodes, which is very useful to screen readers and web crawlers and also helps increase the SEO of your page.

**Review by jonas-45 — COMMENTED** [review]
Changes required:

1. It would be better to use `<ul` and `<li>` to display your menu list
2. Please consider using semantic HTML tags to make your code easily understandable and to also help increase your page SEO ranking

**Comment by jonas-45** [comment]
Also work on your readme file to make it more professional. Use this template as a guide https://github.com/microverseinc/readme-template

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @jonas-45,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Professional and clear comments. ✔️
 - Good explanation of why to do the changes. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**